.env-sample

```diff
@@ -1,19 +1,15 @@
-CONTENTFUL_SPACE_ID=
-CONTENTFUL_ENV=
-CONTENTFUL_MANAGEMENT_TOKEN=
-CONTENTFUL_ACCESS_TOKEN=
-CRDS_APP_CLIENT_ENDPOINT=
-CRDS_CMS_SERVER_ENDPOINT=
-PUSHPAY_GIVE_ENDPOINT=
-IMGIX_SRC=
-IMGIX_DOMAIN=
-STREAM_SCHEDULE_ENDPOINT=https://8k97vbzbrk.execute-api.us-east-1.amazonaws.com/int/streamSchedule
-CRDS_CMS_ENDPOINT=https://contentint.crossroads.net
-CRDS_GATEWAY_ENDPOINT=https://gatewayint.crossroads.net/gateway
-CRDS_MEDIA_ENDPOINT=https://mediaint.crossroads.net
-CRDS_SEARCH_ENDPOINT=https://int-crds-search.netlify.com
-AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY=
-AWS_REGION=us-east-1
-LOGZIO_API_KEY=
-APPLICATION_NAME=
+export CONTENTFUL_ACCESS_TOKEN=8cd3539d1e0bf1a92c9b544c194cde662a5d00e20aa85c3bf98949e3aa2650c5
```

Comment:
hola, did you mean to modify this file? Feels like maybe this was meant to be a local change only?

Reply:
@christiancrawford definitely a mistake. Thank you!